packages/common/src/Adaptor.js

```diff
@@ -67,6 +67,27 @@ export function fn(func) {
   };
 }
 
+/**
+ * A custom step (or operation) that will only execute if the condition returns a truthy value
+ * @public
+ * @example
+ * fnIf((state) => state?.data?.name, get("https://example.com"));
+ * @function
+ * @param {Boolean} truthyCondition is the condition that returns true
```

Comment:
I would just call it `condition`, and the docs  are something like `execute the function if this condition returns true`